docker-compose.yml

```diff
@@ -1,14 +1,35 @@
 version: '3'
 services:
```

Comment:
don't need to worry about commenting this out if you scale the container down 
`docker-compose up --scale help-with-prison-visits-internal=0`